### PR TITLE
Dj/update/sd prompts

### PR DESCRIPTION
**Pull request by djohn06**
Updated SD Prompts, more can be added by anyone who wants to!

### CODE REVIEW

**Comment by toppy007** [comment]
nice work danny

**Review by toppy007 — COMMENTED** [review]
just some className conflicts but we can have a look together

**Review by toppy007 — APPROVED** [review]

**Comment by umutbaykan** [comment]
once the class names are resolved, we can look into importing the available prompts from the prompt bank inside the front end folder.
this way we get around the issue of updating two separate lists as we currently have to hardcode the options on front end to reflect the available options on the back end prompt bank.
give me a shout when you guys are through the merge conflicts and lets have a look

**Review by jonpillay — APPROVED** [review]
Looking good. 👍

File: frontend/src/components/form-container/FormContainer.js [hunk 1]
```diff
@@ -81,7 +74,7 @@ const FormContainer = ({ navigate }) => {
             }
           />
           <Form
-            dropdownItems={genre}
+            dropdownItems={genres}
```
Comment (umutbaykan) [inline frontend/src/components/form-container/FormContainer.js:77]: needs to be singular

**Pull request merged**